frontend/vue.config.js

```diff
@@ -15,6 +15,7 @@ module.exports = {
     },
   },
   lintOnSave: true,
+  publicPath: '/vue/',
```

Comment:
this should be an env variable since this might change according to setup. In the long run it will be hosted on the main domain. Furthermore a sensitive default for development would be `/`